docker/server/Dockerfile

```
@@ -0,0 +1,30 @@
+ARG SERVER_NAME
+
+FROM node:20.12.2 AS base
+COPY . /usr/src/app
+
+
+FROM base AS builder
+ARG SERVER_NAME
+WORKDIR /usr/src/app
+RUN yarn dlx turbo@^1.12.5 prune ${SERVER_NAME} --docker
+
+
+FROM node:20.12.2 AS installer
+WORKDIR /usr/src/app
+COPY --from=builder /usr/src/app/out/json/ .
+RUN yarn install --immutable
+COPY --from=builder /usr/src/app/out/full/ .
+RUN yarn turbo run build
+
+
+FROM node:20.12.2-alpine AS runner
+ARG SERVER_NAME
+WORKDIR /usr/src/app
+COPY --from=installer /usr/src/app/apps/${SERVER_NAME} .
```

I suggest to add this to prevent permission errors. for example, executing `io-session-manager`, I encountered permission errors for `certs` dir
```suggestion
COPY --from=installer --chown=node:node /usr/src/app/apps/${SERVER_NAME} .
```